### PR TITLE
feat(migrate): Allow migration engine to log to stdout

### PR DESCRIPTION
**Pull request by tomhoule**
As a first step, we only log what migrations are being applied.

See the corresponding engines PR at https://github.com/prisma/prisma-engines/pull/2044

closes https://github.com/prisma/migrations-team/issues/241
closes https://github.com/prisma/migrations-team/issues/220

### CODE REVIEW

**Comment by Jolg42** [comment]
Note: Please don't merge before engines' PR is merged.

**Comment by millsp** [comment]
What's the status of this?

**Comment by tomhoule** [comment]
Once this PR is reviewed and approved, I'll merge the engines PR.

**Comment by Jolg42** [comment]
@tomhoule I cannot review/QA before there is an engine released on integration actually

**Comment by tomhoule** [comment]
(engines PR merged 2 days ago)

**Comment by Jolg42** [comment]
@tomhoule can you merge master in here?

**Comment by tomhoule** [comment]
done

File: packages/migrate/src/MigrateEngine.ts [hunk 1]
```diff
@@ -275,6 +275,10 @@ export class MigrateEngine {
           try {
             const json: MigrateEngineLogLine = JSON.parse(data)
 
+            if (json.fields?.migrate_action === 'log') {
+              console.log(json.fields.message)
```
Comment (Jolg42) [inline packages/migrate/src/MigrateEngine.ts:279]:
```suggestion
              console.info(json.fields.message)
```

**Comment by Jolg42** [comment]
Will be done in https://github.com/prisma/prisma/pull/8536

**Pull request merged**